src/librustc_back/x86.rs

```diff
@@ -45,10 +45,7 @@ pub fn get_target_strs(target_triple: String, target_os: abi::Os)
             "e-p:32:32-f64:32:64-i64:32:64-f80:32:32-n8:16:32".to_string()
           }
 
-          abi::OsFreebsd => {
-            "e-p:32:32-f64:32:64-i64:32:64-f80:32:32-n8:16:32".to_string()
-          }
-          abi::OsDragonfly => {
+          abi::OsFreebsd | abi::OsDragonfly | abi::OsBitrig => {
             "e-p:32:32-f64:32:64-i64:32:64-f80:32:32-n8:16:32".to_string()
           }
```

Comment:
If you collapse `abi::OsFreebsd` + `abi::OsDragonfly` with `abi::OsBitrig`, could you include also `abi::OsOpenbsd` which is just under ? Thanks.